village/src/App.js

```diff
@@ -11,14 +16,77 @@ class App extends Component {
       smurfs: [],
     };
   }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:3333/smurfs')
+      .then(response => {
+        console.log(response)
+        this.setState({
+          smurfs: response.data
+        })
+      })
+      .catch(err => console.log(err))
+  }
+
+  addNewSmurfs = (smurf) => {
+    axios
+      .post('http://localhost:3333/smurfs', smurf)
+      .then(response => {
+        console.log(response)
+        this.setState({
+          smurfs: response.data
+        })
+      })
+      .catch(err => { console.log(err) })
+
+  }
+
+  deleteSmurf = (id) => {
+    axios
+    .delete(`http://localhost:3333/smurfs/${id}`)
+    .then(res => {
+      this.setState({
+        smurfs: res.data
+      })
+    })
+    .catch(err => console.log(err))
+  }
+
+
+
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
+    console.log(this.state)
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        
+            <ul className="nav">
+              <li>
+              <NavLink to="/">
+                  Home
+                </NavLink>
+              </li>
+              <li>
+                <NavLink to="/smurfs">
+                  Smurf Village
+                </NavLink>
```

Comment:
With all the styling that you implemented, I would have loved to see some NavLink Active Class Name styling implemented. Let me know if you have any questions about that, because it's a very cool feature of NavLinks.